src/python/pants/backend/awslambda/common/awslambda_common_rules.py

```diff
@@ -60,7 +60,7 @@ class AWSLambdaGoal(Goal):
             DirectoryToMaterialize(merged_digest, path_prefix=str(distdir.relpath))
         )
         for path in result.output_paths:
-            print_stdout(f"Wrote {path}")
+            print_stdout(f"Wrote {os.path.relpath(path)}")
```

Comment:
Should request the `BuildRoot` singleton, and relativize to that. Example: https://github.com/pantsbuild/pants/blob/2d6853ce915048504cc9ed31222aedd69b7bf588/src/python/pants/rules/core/distdir.py#L24-L38